src/Files.Uwp/UserControls/Menus/ColoredToggleMenuFlyoutItem.xaml

```diff
@@ -0,0 +1,183 @@
+﻿<ToggleMenuFlyoutItem
+    x:Class="Files.Uwp.UserControls.Menus.ColoredToggleMenuFlyoutItem"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:local="using:Files.Uwp.UserControls.Menus"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:vc="using:Files.Uwp.ValueConverters"
+    d:DesignHeight="300"
+    d:DesignWidth="400"
+    mc:Ignorable="d">
+
+    <ToggleMenuFlyoutItem.Resources>
+        <vc:ColorModelToColorConverter x:Key="ColorModelToColorConverter" />
+    </ToggleMenuFlyoutItem.Resources>
+
+    <ToggleMenuFlyoutItem.Template>
+        <ControlTemplate TargetType="local:ColoredToggleMenuFlyoutItem">
+            <Grid
+                x:Name="LayoutRoot"
+                Padding="{TemplateBinding Padding}"
+                Background="{TemplateBinding Background}"
+                BorderBrush="{TemplateBinding BorderBrush}"
+                BorderThickness="{TemplateBinding BorderThickness}"
+                CornerRadius="{TemplateBinding CornerRadius}">
+                <Grid x:Name="AnimationRoot">
+
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition Width="Auto" />
+                        <ColumnDefinition Width="*" />
+                        <ColumnDefinition Width="Auto" />
+                    </Grid.ColumnDefinitions>
+                    <Border
```

Comment:
The round corners are a little blurred, can you try using an ellipse?

Reply:
Done, is it any better?

Reply:
Much better